tests/fast-integration/package.json

```diff
@@ -14,6 +14,9 @@
     "test-no-install": "mocha --inline-diffs --check-leaks -g Installation --invert --reporter mocha-multi-reporters --reporter-options configFile=mocha-reporter-config.json",
     "test-utils": "mocha --inline-diffs --check-leaks --reporter mocha-multi-reporters --reporter-options configFile=mocha-reporter-config.json './utils/**/*.test.js'",
     "test-commerce-mock": "mocha --inline-diffs --check-leaks --reporter mocha-multi-reporters --reporter-options configFile=mocha-reporter-config.json ./test/2-commerce-mock.js",
+    "test-commerce-mock-prep-and-tests": "mocha --inline-diffs --check-leaks --reporter mocha-multi-reporters --reporter-options configFile=mocha-reporter-config.json ./test/2-commerce-mock-prep-and-tests.js",
```

Comment:
why not having `test-commerce-mock-prep` only, this way you would have in makefile something like:
```
npm run test-commerce-mock-prep
npm run test-commerce-mock-tests
```
i would be clean and simple

Reply:
At first this is what was done. But for some reason tests were timeout'ing so I grouped up these two steps. Now I came back to original approach and fortunately it works. 👍